### ISSUE TITLE
Liquibase-Hibernate does not support JDK 1.6

### ISSUE THREAD

**Issue by nils-christian** [liquibase/liquibase-hibernate#158]
According to the project's wiki page (https://github.com/liquibase/liquibase-hibernate/wiki), the extension requires Java 1.6+. However, at least Version 3.6 (I did not check the others) does not support JDK 1.6.

One can check this with the javap tool.

`javap -verbose HibernateConnection.class | findStr "major"`

leads to

`major version: 51`

Major version 51 indicates Java 7. The library should either be compiled to be fully Java 6 compatible or the wiki entry should be corrected.

┆Issue is synchronized with this [Jira Bug](https://datical.atlassian.net/browse/LB-464) by [Unito](https://www.unito.io/learn-more)

**Comment by nils-christian** [comment]
In the meantime, I checked some things:
* The issue seems to be the case only for Version 3.6. I took some random samples from 3.5 and 3.4 and couldn't find any Java 7 classes (supplied without liability).
* To make things worse, Version 3.6 of Liquibase-Hibernate has H2 as dependency. According to the project's page, the H2 database requires Java 7 as well. A quick check confirms that H2 is compiled with JDK 7.
